src/tpm.rs

```diff
@@ -525,6 +591,54 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_get_tpm_metadata_1() {
+        match set_tpmdata_test() {
+            Ok(_) => {
+                // using test tpmdata.json content must present, system won't panic
```

Comment:
Instead of panicing, consider printing a useful error message.

Reply:
Print out an error message explaining what happened.